src/legacy/core_plugins/kibana/public/management/sections/index_patterns/edit_index_pattern/tabs/tabs.tsx

```diff
@@ -33,6 +33,7 @@ import { i18n } from '@kbn/i18n';
 import { fieldWildcardMatcher } from '../../../../../../../../../plugins/kibana_utils/public';
 import { IndexPatternManagementStart } from '../../../../../../../../../plugins/index_pattern_management/public';
 import { IndexPattern, IndexPatternField } from '../../../../../../../../../plugins/data/public';
+import { META_FIELDS_SETTING } from '../../../../../../../../../plugins/data/common';
```

Comment:
If I'm not wrong we are trying to avoid imports from /`common` folder. 

Also I think that this approach introduce a lot of fields in our static contract which also not good.

Maybe we can move all into one variable: 

```
const UI_SETTINGS = {
  META_FIELDS_SETTING: 'meta_field',
  DEFAULT_QUERY_LANGUAGE: 'kuery',
  META_FIELDS_SETTING: `'metaFields',`
}
```

Reply:
sorry, seems we had a comment/merge race condition. seen this comment after the merge. since this part of Kibana was in the legacy world, and it's currently migrated, this could be done in or after #65026